scripts/build-plugins/rollup-plugin-build-themes.js

```diff
@@ -106,18 +143,57 @@ module.exports = function buildThemes(options) {
             }
         },
 
+        resolveId(id) {
+            if (id.startsWith(virtualModuleId)) {
+                return isDevelopment? '\0' + id: false;
+            }
+        },
+
         async load(id) {
-            const result = id.match(/(.+)\/theme.css\?variant=(.+)/);
-            if (result) {
-                const [, location, variant] = result;
-                const cssSource = await readCSSSource(location);
-                const config = variants[variant];
-                return await appendVariablesToCSS(config.variables, cssSource);
+            if (isDevelopment) {
+                if (id.startsWith(resolvedVirtualModuleId)) {
+                    let [theme, variant, file] = id.substr(resolvedVirtualModuleId.length).split("/");
+                    if (theme === "default") {
+                        theme = "Element";
+                    }
+                    if (!variant || variant === "default") {
+                        variant = "light";
+                    }
+                    if (!file) {
+                        file = "index.js";
+                    }
+                    switch (file) {
+                        case "index.js": {
```

Comment:
can we add a comment somewhere why we have to take this different approach for both dev and build mode, e.g. that to be able to have the theme css file pass the normal css pipeline, going through postcss, we can't dynamically return it from our plugin (I think that is the reason?). Also, something something the generate bundle hook isn't supported on dev mode, can't remember the full reason now?